specs/interop/optimism-portal-interop.md

```diff
@@ -0,0 +1,49 @@
+# OptimismPortal
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Interface and properties](#interface-and-properties)
+  - [Integrating `SharedLockbox`](#integrating-sharedlockbox)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The `OptimismPortal` contract is upgraded to integrate the `SharedLockbox` and start using the shared ETH liquidity.
+
+### Interface and properties
+
+The `OptimismPortal` contract will add the following storage layout and modified functions:
+
+**`SHARED_LOCKBOX`**
+
+- An immutable address pointing to the `SharedLockbox` contract.
+- This address MUST be immutable because all `OptimismPortals` will point to the same `SharedLockbox`,
```

Comment:
Immutable but not a immutable variable

Reply:
The value should never change but if we make it an `immutable` in practice, it means that chains that upgrade but don't join the cluster instantly will need to deploy their own portal contract, which just adds cost. Another way to do it would be to be the address of the lockbox on the superchain config and fetch it from there, it adds an extra call which isn't ideal but it does reduce the amount of config footguns